src/model/element.ts

```diff
@@ -32,13 +33,14 @@ export class ScannedElement implements Resolvable {
   className?: string;
   superClass?: string;
   extends?: string;
-  properties: ScannedProperty[] = [];
-  attributes: ScannedAttribute[] = [];
+  properties = [] as ScannedProperty[];
```

Comment:
I think it's a toss-up here. One is a type annotation to fill in a gap in type inference, the other is a cast, which then makes type inference work. Here, I think I prefer the type annotation so that the list of variables and their types is relatively consistent. It's also good to have a consistent `let or const` `name` `type` `initializer` ordering.